pkgs/servers/home-assistant/custom-components/hass-scene_presets/disable-custom-presets.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/custom_components/scene_presets/file_utils.py b/custom_components/scene_presets/file_utils.py
+index 3658fd1..da799c2 100644
+--- a/custom_components/scene_presets/file_utils.py
++++ b/custom_components/scene_presets/file_utils.py
+@@ -12,7 +12,7 @@ VERSION = MANIFEST['version']
+ 
+ 
+ 
+-os.makedirs(os.path.join(BASE_PATH, 'userdata/custom/assets'), exist_ok=True)
++# os.makedirs(os.path.join(BASE_PATH, 'userdata/custom/assets'), exist_ok=True)
```

Comment:
Can we rather change where those custom presets and assets are stored?
It would be worth opening an upstream issue.

Reply:
We could, but I'm not fussy about it for now as I'm happy with the wide variety of presets already available.